styles.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-position: 0.1rem 1.5cm;
+  background-repeat: no-repeat;
+  background-image: url(/img/background.svg);
```

Comment:
- [ ] Kindly add this background image to the headline section the one you give a class named `content` instead of adding it to the whole body.

- [ ] Please add the correct image path by adding a `.` at the beginning of the current path because now It's not working online. 

![image](https://user-images.githubusercontent.com/68796232/185284592-0d207156-5863-40aa-af01-ba24467cea0e.png)